bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tdirectory.py

```diff
@@ -19,6 +19,6 @@
 @pythonization(lazy = False)
 def pythonize_tdirectory():
     klass = cppyy.gbl.TDirectory
-    AddDirectoryAttrSyntaxPyz(klass)
+    AddTDirectoryGetAttr(klass)
```

Comment:
To keep the nomenclature, maybe `AddTDirectoryGetAttrPyz` ?

Reply:
Actually, it should be `AddDirectoryGetAttrPyz`, since there are others like `AddFile...` but not `AddTFile`.